### ISSUE TITLE
Keyup triggered on non-configured keys

### ISSUE THREAD

**Issue by tpott** [BashVideo/google-meet-push-to-talk#4]
Hey! Thanks for putting this extension together. I've been testing it out some today and really like the overall experience. 

I encountered a bug where changing applications or tabs and coming back to the Meet causes the `keyup` event to trigger for other keys. For example, I configured my hot key to the option key (⌥ on a Mac), which maps to Alt for javascript event codes. When I switch to another application from Chrome, and then come back, I get muted. 

From adding some console.log statements, and trying to figure out what's going on, I think the problem is that `matchKeyup` in src/js/hotkey.js is matching on non-configured keys. From the above example, I logged that `this.keys.altKey` is true, but it seems like the keyup event from the command key (⌘ on a Mac; maps to the MetaKey) has `event.altKey` set to false, which causes `matchKeyup` to return true.

**Comment by stopdropandrew** [comment]
This is fantastic, thanks for the pull! I'm pretty swamped at work this week, but I'll check it out this weekend. 🙌🏼

**Comment by tpott** [comment]
Sweet! I'll check back this weekend then :)

**Comment by tpott** [comment]
Pinging this just in case this drifted off

**Comment by stopdropandrew** [comment]
Hi there, sorry about the delay. After testing this out I remembered why I set it up this way in the first place. The reason is if you try a key combination like "shift-D" for instance, you press shift and D and are unmuted. If you release *either* shift OR D you are muted again.

With your pull it changes to only muting when you let go of the D.

I found another solution #8 by tracking whether a full keypress has happened. It should fix the issue of triggering keyups too easily. Try it out and if it works for you I'll merge it. Thanks again!
